packages/elements/src/utils/duplicateElement.ts

```diff
@@ -33,7 +33,7 @@ import createNewElement from './createNewElement';
  * @param originY Original Y.
  * @return Coordinates.
  */
-function getOffsetCoordinates(originX: number, originY: number) {
+export function getOffsetCoordinates(originX: number, originY: number) {
```

Comment:
Now that this is exported, should we move it to its own `getOffsetCoordinates.ts` file for visibility & maintenance?